src/interactivity/win32/Clipboard.cpp

```diff
@@ -319,6 +300,40 @@ void Clipboard::CopyTextToSystemClipboard(const TextBuffer::TextAndColor& rows,
     globalHandle.release();
 }
 
+// Routine Description:
+// - Copies the given string onto the global system clipboard in the sepcified format
+// Arguments:
+// - stringToCopy - The string to copy
+// - lpszFormat - the name of the format
+void Clipboard::CopyToSystemClipboard(std::string stringToCopy, LPCWSTR lpszFormat)
+{
+    const size_t rtfSize = stringToCopy.size() + 1; // +1 for '\0'
+    if (rtfSize)
+    {
+        wil::unique_hglobal globalHandleHTML(GlobalAlloc(GMEM_MOVEABLE | GMEM_DDESHARE, rtfSize));

```

Comment:
I'd maybe rename this to be more generic
```suggestion
        wil::unique_hglobal globalHandleData(GlobalAlloc(GMEM_MOVEABLE | GMEM_DDESHARE, rtfSize));
```

Reply:
Done